src/pages/Auth/signup.jsx

```diff
@@ -0,0 +1,8 @@
+import { SignupBody } from "../../component/SignupBody"
+
+export const Signup = ()=>{
+   return( <>
+        <SignupBody/>
+   
+    </>)
```

Comment:
suggestion: unless you need this file for additional jsx, better to use `<SignupBody />` directly after renaming it to `<Signup />`